chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -134,6 +135,7 @@ spec:
 {{ toYaml .Values.scheduler.resources | indent 12 }}
           image: {{ template "airflow_image_for_migrations" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          securityContext: {{ $containerSecurityContext | nindent 12 }}
```

Comment:
Shouldn't this use `scheduler.waitForMigrations.securityContexts.container`?

Reply:
If you'd rather not do it for every single container everywhere in this PR, that's fine with me. It can be expanded in a followup PR.